src/sentry/static/sentry/app/views/settings/components/settingsNavigation.tsx

```diff
@@ -1,24 +1,31 @@
 import * as Sentry from '@sentry/browser';
-import {Box} from 'grid-emotion';
-import PropTypes from 'prop-types';
 import React from 'react';
 
 import SettingsNavigationGroup from 'app/views/settings/components/settingsNavigationGroup';
-import SentryTypes from 'app/sentryTypes';
+import {NavigationSection, NavigationProps} from 'app/views/settings/types';
 
-class SettingsNavigation extends React.Component {
-  static propTypes = {
```

Comment:
@EvanPurkhiser you can always leave the `propTypes` and type it as `any`. Until we are 100% on TS, I think we should keep `propTypes` if they already exist.

Reply:
This isn't used by anything that isn't typescript :)

Reply:
@EvanPurkhiser yea...I guess someone could make a new JSX file that uses it but I suppose that's unlikely.